common/log/zap_logger.go

```diff
@@ -216,13 +216,17 @@ func buildZapLogger(cfg Config, disableCaller bool) *zap.Logger {
 		encodeConfig.EncodeCaller = nil
 	}
 
-	outputPath := "stderr"
+	var outputPaths []string
 	if len(cfg.OutputFile) > 0 {
-		outputPath = cfg.OutputFile
+		outputPaths = append(outputPaths, cfg.OutputFile)
 	}
 	if cfg.Stdout {
-		outputPath = "stdout"
+		outputPaths = append(outputPaths, "stdout")
 	}
+	if len(outputPaths) == 0 {
```

Comment:
What about file + stderr?

Reply:
I believe stderr is that default outputPath used if there is no temporal log configuration. Do we want to introduce a config for stderr as well?

Reply:
I think we should just leave it as is and not take both. We can document the behavior so it's not surprising and if anyone needs both for some reason, they can use `| tee` when starting the server.